### DISCUSSION TITLE
Pandoc installation problem

### DISCUSSION THREAD

**Discussion by joacorapela** [community/community#36334] [Actions]
I have a Github action to build my Sphinx docs. The Sphinx docs include a Jupyter (*.ipynb) notebook. Locally the documentation builds fine, but the Github action fails with the following message:

> Notebook error:
> PandocMissing in ipynb/doEstimateAndPlot_defaultInitialParams.ipynb:
> Pandoc wasn't found.
> Please check that pandoc is installed:
> https://pandoc.org/installing.html
> make: *** [Makefile:22: html] Error 2

This is the link to the workflow:
https://github.com/joacorapela/svGPFA/actions/workflows/docs.yml

and this is a link to an example run showing the previous error message:
https://github.com/joacorapela/svGPFA/actions/runs/3260313559

GitHub Action workflow:
> name: docs
> on: [push, pull_request, workflow_dispatch]
> jobs:
>   docs:
>     runs-on: ubuntu-latest
>     steps:
>       - uses: actions/checkout@v2
>       - uses: actions/setup-python@v2
>       - name: Install Pandoc
>         uses: r-lib/actions/setup-pandoc@9598b8eeb6d88de7d76d580d84443542bbfdffce
>         with:
>           pandoc-version: 2.11.1.1
>       - name: Install dependencies
>         run: |
>           pip install -e .[notebook]
>       - uses: ammaraskar/sphinx-action@master
>         with:
>           docs-folder: "docs/sphinx"
>       - name: Deploy
>         uses: peaceiris/actions-gh-pages@v3
>         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
>         with:
>           publish_branch: gh-pages
>           github_token: ${{ secrets.GITHUB_TOKEN }}
>           publish_dir: docs/sphinx/build/html
>           force_orphan: true

It seems that the Pandoc installation in step ``Install Pandoc`` succeeds:
> Preparing to unpack .../pandoc-2.11.1.1-1-amd64.deb ...
> Unpacking pandoc (2.11.1.1-1) ...
> Setting up pandoc (2.11.1.1-1) ...
> Processing triggers for man-db (2.9.1-1) ...

**Comment by ossanna16** [comment]
Hey there @joacorapela and thank you for asking a good question! I'm glad you found our community 🙂
